docs/using_eessi/eessi_container.md

```diff
@@ -0,0 +1,165 @@
+```
+Just try the following command to "resume" from the last session. _Note, the
+parameter after `--resume` is very likely different for your last session. Just
+scroll back and take note of it._
+```
+./eessi_container.sh --resume /tmp/eessi.RC6kYEjmSj
+```
+This should run much faster because the container image has been cached in the
+temporary directory (`/tmp/eessi.RC6kYEjmSj` in the example above). You should
+get to the prompt (`Singularity > ` or `Apptainer > `). Now you just have
+to source again the EESSI init script by running
+```
+source /cvmfs/pilot.eessi-hpc.org/latest/init/bash
+```
+_(Note, the `/tmp/eessi...` directory contains a `home` directory which includes
+the saved history of your last session. Just type `history` to see it. You should
+be able to access it as you would do in a normal terminal session.)_
+### List available options for `eessi_container.sh`
+Run `./eessi_container.sh --help` to see a list of available options.
+```
+./eessi_container.sh --help
+usage: ./eessi_container.sh [OPTIONS] [SCRIPT]
+ OPTIONS:
+  -a | --access {ro,rw} - ro (read-only), rw (read & write) [default: ro]
+  -c | --container IMG  - image file or URL defining the container to use
+                          [default: docker://ghcr.io/eessi/build-node:debian11]
+  -h | --help           - display this usage information [default: false]
+  -g | --storage DIR    - directory space on host machine (used for
+                          temporary data) [default: 1. TMPDIR, 2. /tmp]
+  -m | --mode MODE      - with MODE==shell (launch interactive shell) or
+                          MODE==run (run a script) [default: shell]
+  -r | --repository CFG - configuration file or identifier defining the
+                          repository to use [default: EESSI-pilot via
+                          container configuration]
+  -u | --resume DIR/TGZ - resume a previous run from a directory or tarball,
+                          where DIR points to a previously used tmp directory
+                          (check for output 'Using DIR as tmp ...' of a previous
+                          run) and TGZ is the path to a tarball which is
+                          unpacked the tmp dir stored on the local storage space
+                          (see option --storage above) [default: not set]
+  -s | --save DIR/TGZ   - save contents of tmp directory to a tarball in
+                          directory DIR or provided with the fixed full path TGZ
+                          when a directory is provided, the format of the
+                          tarball's name will be {REPO_ID}-{TIMESTAMP}.tgz
+                          [default: not set]
+  -v | --verbose        - display more information [default: false]
+
+ If value for --mode is 'run', the SCRIPT provided is executed.
+```
+## Running commands or scripts with the container
+### Run a command
+### Run a script
+### Run EESSI demo scripts
+
+## Build a package for `/cvmfs/pilot.eessi-hpc.org`
```

Comment:
Maybe something like `## Getting (local) write access to the EESSI repository`, and then clarify what this means?

Reply:
We should definitely also have a section on save/resume

Reply:
Fine by me to cover `--save` in a later PR